geo/parse.go

```diff
@@ -85,7 +85,7 @@ func ExtractGeoPoint(thing interface{}) (lon, lat float64, success bool) {
 			}
 		} else {
 			// geohash
-			lat, lon = GeoHashDecode(geoStr)
```

Comment:
what happened to this function, shouldn't it be removed?

Reply:
Yes, there is a cleanup pending here for the stale code. But thinking was that lets the new API stand the test of time (the system/longevity tests in CB which are ongoing now for MH) for some time and then I can clean it up..

Reply:
In that case, I would recommend we not do it this way.  Generally, we don't keep unused code in the repository (same reason we don't commit commented out code).  Everyone intends to come back and clean it up, but in reality, we all move on to other tasks and it gets forgotten.

If we're confident enough to use the new function, we should remove the old one.  It's in the git history if we ever need to switch it back.